.gitmodules

```diff
@@ -1,3 +1,7 @@
 [submodule "public/themes"]
 	path = public/themes
 	url = git://github.com/openSUSE/opensuse-themes.git
+
+[submodule "public/chameleon"]
+	path = public/chameleon
+	url = git@github.com:openSUSE-zh/chameleon.git
```

Comment:
ok, is this a generic theme or specific to software.o.o? We could call it opensuse-theme-cameleon for example